src/rviz/CMakeLists.txt

```diff
@@ -7,15 +7,9 @@ aikido_check_package(roscpp "rviz" "roscpp")
 find_package(geometry_msgs QUIET)
 aikido_check_package(geometry_msgs "rviz" "geometry_msgs")
 
-# TODO: Workaround because find_package(interactive_markers) imports "tests"
-# and "run_tests" targets that conflict with the targets defined by Aikido.
-find_path(interactive_markers_INCLUDE_DIRS interactive_markers
-  PATHS "/opt/ros/indigo/include"
-  PATH_SUFFIXES "interactive_markeres/interactive_marker_client.h")
-find_library(interactive_markers_LIBRARIES interactive_markers
-  PATHS "/opt/ros/indigo/lib")
```

Comment:
> interactive_markeres -> interactive_markers, right?

Oh, right. Thanks! interactive_mark**e**res! Yeah, I also believe the path never got used, but leave there just in case with fixed one.